src/Core/Containers/BVH.hpp

```diff
@@ -1,21 +1,32 @@
-
 #ifndef RADIUMENGINE_BVH_HPP
 #define RADIUMENGINE_BVH_HPP
 
 #include <Core/RaCore.hpp>
+#include <Eigen/Geometry>
 
-#include <Core/Math/Frustum.hpp>
-#include <Core/Mesh/TriangleMesh.hpp>
+#include <Core/Geometry/Frustum.hpp>
 
 #include <memory>
 #include <vector>
 
 namespace Ra {
 namespace Core {
-/// This class stores a 3-dimensional hierarchy of meshes of arbitrary type.
-/// Built on a binary tree
+namespace Containers {
+/*!
+ * \brief Stores a 3-dimensional hierarchy of meshes of arbitrary type.
+ *
+ * This class is marked as deprecated as it is not tested nor used.
+ * \FIXME Confirm class status: add tests + use cases *or* remove
+ *
+ * \note Radium might soon have a generic type for geometrical objects.
+ * This would allow to remove the template parameter of this class.
+ *
+ * \note We might consider using this instead:
+ * https://eigen.tuxfamily.org/dox/unsupported/group__BVH__Module.html
+ */
 template <typename T>
-class BVH {
+class [[deprecated( "Neither used nor tested" )]] BVH {
+    using Aabb = Eigen::AlignedBox<Scalar, 3>;
```

Comment:
use it afterwards instead of `Eigen::AlignedBox<Scalar, 3>`

Reply:
Fixed.